src/matchers/matcher/currentDate.matcher.spec.js

```diff
@@ -0,0 +1,63 @@
+import { currentDateMatcher } from './currentDate.matcher';
+import Sugar from 'sugar-date';
+import { expect } from 'chai';
+
+describe('Current Date matcher', () => {
```

Comment:
You're missing tests which covers:
- dates with slash e.g. 2018/03/2018

Reply:
added test for incorrect date